frontend/packages/console-dynamic-plugin-sdk/src/app/core/actions/core.ts

```diff
@@ -1,22 +1,29 @@
 import { action, ActionType as Action } from 'typesafe-actions';
 import { UserInfo } from '../../../extensions';
+import { WarningPolicy } from '../../redux-types';
```

Comment:
Use a more semantically accurate name wherever we reference WarningPolicy or warningPolicy:
```suggestion
import { AdmissionWebhookWarning } from '../../redux-types';
```